parsl/monitoring/router.py

```diff
@@ -200,7 +215,7 @@ def router_starter(comm_q: "queue.Queue[Union[Tuple[int, int], str]]",
 
         router.logger.info("Starting MonitoringRouter in router_starter")
         try:
-            router.start(priority_msgs, node_msgs, block_msgs, resource_msgs, exit_event)
+            router.start()
```

Comment:
Since making this semantic change, I might suggest similarly altering `.start()`, which begins to imply an asynchronous thought-process / operation, to `.run()`.  No functional difference, but at least follows the semantics of a few other libraries (e.g., `Threading.Thread`)

Reply:
... but `multiprocessing` which I guess is what influenced this original uses `.start` and has a `.run` that means something completely different... :exploding_head: